### DISCUSSION TITLE
16:9 aspect ratio

### DISCUSSION THREAD

**Discussion by MontyFlange** [lllyasviel/Fooocus#984] [Ideas]
Can you add a 16:9 aspect ratio ?

**Comment by xhoxye** [comment]
You can customize the aspect ratio ratio in the developer tools, but the results may not be great.
You can also select an original scale and zoom in.
1344: 768 = 7: 4 = 15.75: 9 = 16: 9.143

**Reply by realmalikov** [reply]
1280×768 ∣ 5:3 is even closer to 16:9. Simply remove 24px at the top and bottom, respectively.

**Reply by williamsblood** [reply]
I need some assistance with setting the "Fooocus" to generate larger images without causing distortion. Specifically, I'm trying to use the preset  780x1280 with a scale of 1.5 for image generation. However, when I input 1152x1920, the images turn out distorted... Could you  advise on the ideal pixel proportion for overriding this in developer mode?

**Reply by xhoxye** [reply]
That's why it's recommended to use the recommended ratio of SDXL, which is good for results , and it is difficult to guarantee that the results are good for other ratios

**Reply by williamsblood** [reply]
I'm interested in generating images at a higher resolution because I want to maintain the style of the original image. When I use the standard 1.5 upscale, the image quality improves but often changes significantly from the original. I'm considering overriding or adjusting the denoising feature of the upscale process, but I'm unsure about the ideal parameter settings to keep the upscaled image consistent with the aspect of the original. Could you provide some guidance on this? If I use the 2x fast upscale I keep the original but results are not good.

**Reply by lllyasviel** [reply]
@williamsblood show example when 2x results are not good

**Reply by MontyFlange** [reply]
Cant you make your own thread, This is about aspect ratio

**Reply by williamsblood** [reply]
I did few threads sorry I got no answer  I will post sorry to bother.

**Comment by Fahadbhatti34** [comment]
Can you Add 9:16 Please

**Reply by chouhai2018** [reply]
现在版本,添加后人物会变胖的

**Reply by chouhai2018** [reply]
"available_aspect_ratios": [
		"576 * 1024",
		"1024 * 576",
		"540 * 960",
		"960 * 540",
		"768 * 960",
		"960 * 768",
		"1024 * 819",
		"819 * 1024",
		"1024 * 1280",
		"1280 * 1024",
		"1200 * 960",
		"960 * 1200",
		"512 * 896",
		"896 * 512",
		"768 * 1344",
		"1344 * 768",
		"1024 * 1792",
		"1792 * 1024",
		"1024 * 1024",
        "704 * 1408",
        "704 * 1344",
        "768 * 1280",
        "832 * 1216",
        "832 * 1152",
        "896 * 1152",
        "896 * 1088",
        "960 * 1088",
        "960 * 1024",
        "1024 * 960",
        "1088 * 960",
        "1088 * 896",
        "1152 * 896",
        "1152 * 832",
        "1216 * 832",
        "1280 * 768",
        "1344 * 704",
        "1408 * 704",
        "1472 * 704",
        "1536 * 640",
        "1600 * 640",
        "1664 * 576",
        "1728 * 576"
    ],

**Reply by chouhai2018** [reply]
星号没显示

**Reply by chouhai2018** [reply]
测试过了,不会爆显存

**Comment by DavidDragonsage** [comment]
16:9 is a very convenient aspect ratio - for example its great for making a Facebook banner, as shown. The maximum resolution for SDXL is 1024x1024, or 1,048,576 pixels. As long as you don't run out of resolution in whatever aspect ratio you choose, i.e. as long as the width and height do not create a canvas larger than 1,048,576 pixels in area, you should be good.

I know that theoretically we should not stray from the preset aspect ratios provided in Fooocus. But after a lifetime as an amateur photographer, I'm pretty attached to 2:3 and 3:2 (the 35mm standard) so they were the second aspect ratios I added, after 16:9.

I think I've made some really good images in Fooocus at 2:3 and 3:2. Besides artistry itself - the most important factor - there are many different parameters that go into making a good image but I don't think its wise to obsess too much over any single one of them - especially if its at the expense of your artistic sense and experience.

For reference, 832x1248 and 1248x832 make very good 2:3 and 3:2 aspect ratios, respectively. And I use 1344x756 for 16:9.

There is an excellent tutorial on setting up Fooocus aspect ratios here:
https://stable-diffusion-art.com/fooocus/#Aspect_Ratios

![Pure Fooocus Banner](https://github.com/lllyasviel/Fooocus/assets/162500231/ccef23dc-18ce-464e-aff7-d06841594f55)
